django/contrib/gis/geos/collections.py

```diff
@@ -49,12 +49,11 @@ def __len__(self):
     # ### Methods for compatibility with ListMixin ###
     def _create_collection(self, length, items):
         # Creating the geometry pointer array.
-        geoms = get_pointer_arr(length)
-        for i, g in enumerate(items):
+        geoms = (GEOM_PTR * length)(*[
```

Comment:
Is there a speed advantage or disadvantage to using a tuple vs. list comprehension?

Reply:
```
In [53]: arr = c_long*10

In [54]: %timeit arr(*(x+1 for x in range(10)))
3.74 µs ± 66.1 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)                         

In [55]: %timeit arr(*[x+1 for x in range(10)])
3.41 µs ± 23.9 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)                         

In [56]: %timeit arr(*tuple(x+1 for x in range(10)))
4.18 µs ± 59.9 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```